src/App.jsx

```diff
@@ -103,28 +50,17 @@ function App() {
 						path='/courses'
```

Comment:
It's better to use ProtectedRoute for non public pages - optional
examples here: https://dev.to/olumidesamuel_/implementing-protected-route-and-authentication-in-react-js-3cl4

Reply:
It will be done in the next module